source/manuals/licensing.html.md.erb

```diff
@@ -1,20 +1,34 @@
 ---
 title: Licensing
-last_reviewed_on: 2021-10-12
+last_reviewed_on: 2021-10-19
 review_in: 6 months
 ---
 
 # <%= current_page.data.title %>
 
 ## Guidelines for repositories containing code
 
-Each repository should include a licence file. This should be called `LICENCE` or `LICENCE.md`. "License" is the US English spelling.
+### Language
 
-GitHub.com will still show licence details for the British English spelling.
+* Use the British English spelling of the noun _licence_, not the US English
+  spelling of _license_ (e.g. "you need a licence to drive a car.")
```

Comment:
```suggestion
  spelling of _license_ (for example "you need a licence to drive a car.")
```

Reply:
Ta, I'll change that in both `e.g.` cases.